src/types/module-deployments/CommonModuleMetadata.ts

```diff
@@ -37,4 +37,6 @@ export default class CommonModuleMetadata {
    */
   @JsonProperty("external_link", String, PropertyConvertingMode.IGNORE_NULLABLE)
   externalLink?: string = undefined;
+
+  [key: string]: any;
```

Comment:
What's going on here, are you trying to define the generic `properties` object? Properties aren't really assigned to modules, but we can add it here. Need to make sure we define the property using `json2typescript` like we do the others properties.

Reply:
yep had to do this to make `metadata[key]` work